src/views/OrdersView.vue

```diff
@@ -7,7 +7,17 @@
       :message="toast.message"
       @toastCloseBtnClicked="clearToast">
     </toast-component>
-    <h1 id="orders-title">{{'lifelines-webshop-orders-title' | i18n}}</h1>
+
+    <ConfirmationModal
+      v-if="$route && $route.name === 'orderDelete'"
```

Comment:
seems somewhat strange to use the router for the modal to me

Reply:
I could have used the <router-view/> here, but that would either require to pass all component properties with the route definition, or make a wrapper component for this that does the same. It seemed a bit overkill to me to do the latter.